### PR TITLE
Reaction screen

### PR DESCRIPTION
**Pull request by RaulMorales404**
# Description
What: adds the user reactions screen 
Why:To provide the user with better access to previously viewed sites 
 
  ## Screenshots 
Captura de pantalla-20211104101515-309x656

### CODE REVIEW

File: GoToProject/src/components/CustomIcon/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Icon from 'react-native-ionicons';
+
+export const CustomIcon = ({name,size,color}) => {
+    return(
+        <Icon name={name} style={{fontSize: size, color: color}} />
+    )
+}
```
Comment (YilKanda) [inline GoToProject/src/components/CustomIcon/index.js:8]: Add a line blank.

File: GoToProject/src/containers/Reactions/styled.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components/native';
+import {colors} from '../../library/constants/colors';
+import {TitleText, FlexContainer} from '../../components/styled';
+export const Cart = styled(FlexContainer)`
+  border-radius: 8px;
+`;
+export const Starts = styled.View`
+  background: #59bcf4;
```
Comment (YilKanda) [inline GoToProject/src/containers/Reactions/styled.js:8]: You can import it from your color library.

File: taus [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Date:   Wed Oct 20 20:39:58 2021 -0500
+
+    Redare the documentation
+    
+    The final project documentation
+
+[33mcommit cde3eed9a913cd1ea16fc8d94dd072da57a96371[m
+Author: github-classroom[bot] <66690702+github-classroom[bot]@users.noreply.github.com>
+Date:   Thu Oct 14 15:20:28 2021 +0000
+
```
Comment (YilKanda) [inline taus:130]: What do you need this commit file for?

File: GoToProject/src/components/Reactions/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <Cart h="124px" w="100%" bg={colors.darkGray} dir="row" jc="space-between">
+      <CartImage w="40%" mt="10px">
+        <Image
+          style={{
+            width: 140,
+            height: 124,
+            borderTopLeftRadius: 5,
+            borderBottomLeftRadius: 5,
+          }}
```
Comment (YilKanda) [inline GoToProject/src/components/Reactions/index.js:27]: These styles can be imported from your style file.

File: GoToProject/src/components/CustomIcon/index.js (hunk 1 above)
Comment (RaulMorales404) [reply GoToProject/src/components/CustomIcon/index.js:8]: done

File: GoToProject/src/containers/Reactions/styled.js (hunk 2 above)
Comment (RaulMorales404) [reply GoToProject/src/containers/Reactions/styled.js:8]: done

File: taus (hunk 3 above)
Comment (RaulMorales404) [reply taus:130]: I will remove it, thank you

File: GoToProject/src/components/Reactions/index.js (hunk 4 above)
Comment (RaulMorales404) [reply GoToProject/src/components/Reactions/index.js]: done

**Review by 96vladi — APPROVED** [review]

**Review by EusebioAjas — APPROVED** [review]

**Pull request merged**